app/src/main/java/com/keylesspalace/tusky/fragment/TimelineFragment.java

```diff
@@ -284,6 +284,7 @@ private void tryCache() {
                     }
 
                     this.updateCurrent();
+                    this.loadAbove();
```

Comment:
Now that concurrent refreshes and top loads are possible, lets to them in parallel.